extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -128,6 +118,30 @@ export async function promptImportGithubDatabase(
   return;
 }
 
+export async function askForGitHubRepo(
+  progress: ProgressCallback,
+  suggestedValue?: string,
+): Promise<string | undefined> {
+  progress({
+    message: "Choose repository",
+    step: 1,
+    maxStep: 2,
+  });
```

Comment:
Should we set the progress here? It seems like this would change depending on whether you're fetching a database or creating a skeleton query.